docs/reference/modules/discovery/discovery-settings.asciidoc

```diff
@@ -201,6 +201,12 @@ Sets how long the master node waits for each cluster state update to be
 completely published to all nodes, unless `discovery.type` is set to
 `single-node`. The default value is `30s`. See <<cluster-state-publishing>>.
 
+`cluster.single_node_cluster_seed_hosts_check_interval`::
+(<<static-cluster-setting,Static>>)
+Sets the interval of the check that warns that seed hosts (see
+`discovery.seed_hosts`) have been defined for a fully-formed single-node
+cluster. The default value is `30s`.
```

Comment:
Reflecting on the user-facing view of this feature, I think something more generic would be a bit better. For instance:

```suggestion
`cluster.discovery_configuration_check.interval `::
(<<static-cluster-setting,Static>>)
Sets the interval of some checks that will log warnings about an
incorrect discovery configuration. The default value is `30s`.
```

Not that we necessarily intend to add any more periodic checks in future, but if we did we could use the same schedule.